libsnowflakeclient/lib/error.h

```diff
@@ -17,11 +17,13 @@ extern "C" {
 
 #include <snowflake_client.h>
 
-#define SET_SNOWFLAKE_ERROR(e, ec, m, s) set_snowflake_error(e, ec, m, s, __FILE__, __LINE__)
+#define SET_SNOWFLAKE_ERROR(e, ec, m, sqlstate) set_snowflake_error(e, ec, m, sqlstate, "", __FILE__, __LINE__)
+#define SET_SNOWFLAKE_STMT_ERROR(e, ec, m, sqlstate, uuid) set_snowflake_error(e, ec, m, sqlstate, uuid, __FILE__, __LINE__)
```

Comment:
Two major cases where one doesn't have sfqid provided, and other have sfqid provided.